content/docs/command-reference/plots/modify.md

```diff
@@ -0,0 +1,124 @@
+# plots modify
+
+Modify [plot metrics](/doc/command-reference/plots) by specifying and saving
+plotting options.
+
+## Synopsis
+
+```usage
+usage: dvc plots modify [-h] [-q | -v] [-t [TEMPLATE]] [-x X] [-y Y]
+                        [--no-csv-header] [--title TITLE] [--xlab XLAB]
+                        [--ylab YLAB] [--unset [UNSET [UNSET ...]]]
+
+positional arguments:
+  target                Plot file to set props to.
+```
+
+## Description
+
+It might be not convinient for users as well as automation systems to specify
+all the visualization parameters (such as `y-axis`, `template`, `title` and
```

Comment:
Where are all the viz params listed? Should we call them params or props (as used in the help output)?

Reply:
Possible terms: visualization properties, visualization parameters, display properties, display parameters. I'm not sure what the "correct" one is. I guess it depends on what terminology Vega uses, so maybe @pared knows?

Reply:
> we should use word specification to refer to vega json

Thanks Pawel, https://vega.github.io/vega/docs/specification/ is perfect! I'll link it.

"Vega specification" seems appropriate for the overarching concept, yes (we currently use "schema" sometimes, I'll review.

For what I was asking, it seems they call it "properties" indeed. The visualization vs. display part we should decide.

Reply:
> The visualization vs. display part we should decide.

I'm going to go for "display properties". Visualization can be used for more general concepts.

Reply:
OK all done now, thanks! Resolving.